index.js

```diff
@@ -53,15 +53,16 @@ function wrapArgs(fns, isParam) {
  */
 
 function wrap(fn, isParam) {
-  const isErrorHandler = !isParam && fn.length == 4;
-  async function wrapped() {
-    next = _once(arguments[2 + isErrorHandler]);
```

Comment:
I guess the main problem was that `next` and `res` are not const and therefore global/rewriting itself